definitions/tools/varscan_somatic.cwl

```diff
@@ -21,7 +21,10 @@ inputs:
             position: 2
         secondaryFiles: [^.bai]
     reference:
-        type: string
+        type:
+            - string
+            - File
+        secondaryFiles: [.fai, ^.dict]
```

Comment:
Do all of these actually need the `.dict` file?  I don't relish the thought of digging through all 80-something of these to verify that, but I'm pretty sure, for example, that varscan here does not.